src/containers/Sidebar/index.js

```diff
@@ -187,21 +158,27 @@ export const Sidebar = ({ sidebarSize = 'tight' }) => {
               onChange=${setSearchValue}
             />
 
-            <${FoldersWrapper} data-testid="sidebar-folders-container">
-              ${folders.map(
-                (folder) =>
-                  html`<${SidebarNestedFolders}
-                    item=${folder}
-                    testId=${`sidebar-folder-${folder.id}`}
-                    onFolderExpandToggle=${handleFolderExpandToggle}
-                    key="rootFolder"
-                  />`
-              )}
+            <${FoldersWrapper}>
+              ${folders.map((folder) => {
```

Comment:
please destructure keys from folder

```suggestion
       ${folders.map(({
            id,
            isRoot,
            name,
            icon,
            isActive
        }) => {
```